graphql-dgs-codegen-core/src/test/resources/kotlin2/constantsForInputTypes/expected/types/Person.kt

```diff
@@ -12,7 +12,7 @@ import kotlin.String
 @JsonDeserialize(builder = Person.Builder::class)
 public class Person(
   firstname: () -> String? = firstnameDefault,
-  lastname: () -> String? = lastnameDefault
+  lastname: () -> String? = lastnameDefault,
```

Comment:
Kotlin Poet will now add a `,` to all constructor arguments. This caused several tests that were matching the expected code with the generated using string comparison.